Error: Cannot find module './normalized-options'

I just installed this command line tool:

```
yarn global add create-react-native-module
```

Everything seems fine.

However, trying to execute any command fails:

```
$ create-react-native-module -V
internal/modules/cjs/loader.js:657
    throw err;
    ^

Error: Cannot find module './normalized-options'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:655:15)
    at Function.Module._load (internal/modules/cjs/loader.js:580:25)
    at Module.require (internal/modules/cjs/loader.js:711:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (~/.config/yarn/global/node_modules/create-react-native-module/command.js:3:27)
    at Module._compile (internal/modules/cjs/loader.js:805:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:816:10)
    at Module.load (internal/modules/cjs/loader.js:672:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:612:12)
    at Function.Module._load (internal/modules/cjs/loader.js:604:3)
```

--- a/package.json
+++ b/package.json
@@ -31,6 +31,7 @@
     "cli.js",
     "command.js",
     "lib.js",
+    "normalized-options.js",
     "README.md"
   ],
   "author": "Christopher J. Brody <chris@brody.consulting>",

